poly-synth/poly-synth.html

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Rename this file to `index.html` so visiting to the URL `poly-synth/` will immediately load the page.